tests/TestableIO.System.IO.Abstractions.TestingHelpers.Tests/TestableIO.System.IO.Abstractions.TestingHelpers.Tests.csproj

```diff
@@ -24,8 +24,7 @@
     <EmbeddedResource Include="TestFiles\TestFile.txt" />
   </ItemGroup>
   <ItemGroup>
-    <ProjectReference Include="../../src/System.IO.Abstractions/System.IO.Abstractions.csproj" />
-    <ProjectReference Include="../../src/System.IO.Abstractions.TestingHelpers/System.IO.Abstractions.TestingHelpers.csproj" />
+    <ProjectReference Include="../../src/TestableIO.System.IO.Abstractions.TestingHelpers/TestableIO.System.IO.Abstractions.TestingHelpers.csproj" />
```

Comment:
`TestableIO.System.IO.Abstractions` is pulled in transitively, so I removed the direct reference.

Reply:
I was wondering, why the build system succeeded, despite this typo. This is even better!